changelog/unreleased/issue-20223.toml

```diff
@@ -0,0 +1,5 @@
+type="a"
```

Comment:
Can this file please be renamed to `pr-20236.toml`? The general change log guidelines suggest using the PR file format when a PR is available.

Reply:
`Only use the pr- name format if there is no related issue for a change in a pull request or if the issue is from a different repository.`

I think issue is good here

Reply:
> Can this file please be renamed to `pr-20236.toml`? The general change log guidelines suggest using the PR file format when a PR is available.

@Nithin-Kasam Apologies, I was incorrect about this, and it was actually correct the way you had it before. The `issue-#` format is indeed preferred if an issue is available.